### PR TITLE
Add symlink for custom fonts

### PR DESCRIPTION
**Pull request by jdnichollsc**
Added:
- Symlink for custom fonts
- Example using Ionicons from web and nativescript projects

### CODE REVIEW

**Comment by sean-perkins** [comment]
Can you update this PR to only contain the symlink changes for linking the font directories between web and native? The goal of this seed is to eliminate extra bloat or dependencies that most users may not need in their own projects. As many users will most likely not need Ionic's icon set, it doesn't make sense to ship it with the main seed.

**Comment by jdnichollsc** [comment]
@sean-perkins Done! 👍

File: src/app.scss [hunk 1]
```diff
@@ -1,12 +0,0 @@
-/*
```
Comment (jdnichollsc) [inline src/app.scss]: @sean-perkins this file is not being used in the app

Comment (sean-perkins) [reply src/app.scss]: We should include the base stylesheet though, as many users will leverage the NativeScript Theme for their projects. This file is used for any global styles you want in your NS application.

Comment (jdnichollsc) [reply src/app.scss]: I was testing creating styles in that file but it doesn't works

Comment (sean-perkins) [reply src/app.scss]: Ahh yeah, we haven't circled back to update that portion. In our own version we have our app component stylesheet reference that file:
```
@import '../app.scss'
```

and updated the view encapsulation to allow it to persist globally:
```
@Component({
    moduleId: module.id,
    selector: 'loop-app',
    templateUrl: './app.component.html',
    styleUrls: ['./app.component.scss'],
    encapsulation: ViewEncapsulation.Native
})
export class AppComponent {
```

It's an issue with webpack where it doesn't seem to recompile that sass file overtop the `app.css` file, which has this:

```
/** Temporary File for Webpack until we figure out the app.css conflict **/
```

I can see the confusion of leaving that file, as users would expect it to apply global styling out of the box. We can remove it and see what people say after.

**Pull request merged**